__index__.md

```diff
@@ -1,8 +1,10 @@
 
 - **[cache/v0.1](/cache/v0.1)** ([📄 graphql](cache/v0.1/cache-v0.1.graphql))
+- **[cache/v0.2](/cache/v0.2)** ([📄 graphql](cache/v0.2/cache-v0.2.graphql))
```

Comment:
Intentional add here for this PR?

Reply:
I'm not sure how this work, this was added by running `npm run build`. I'm guessing this was just an oversight of a previous PR but not familiar enough with the build toolchain to be 100% sure 🤷